gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/BundleMapperService.java

```diff
@@ -65,13 +81,21 @@ public Bundle mapToBundle(RCMRIN030000UK06Message xmlMessage) {
         var referralRequests = mapReferralRequests(ehrFolder, patient);
         addEntries(bundle, referralRequests);
 
-        var observations = mapObservations(ehrExtract, patient, List.of()); //TODO: Provide list of encounters
+        var observations = mapObservations(ehrExtract, patient, encounters);
         addEntries(bundle, observations);
 
+        // TODO: Add references to mapped resources in their appropriate lists
```

Comment:
Do we have a ticket for that?

Reply:
https://gpitbjss.atlassian.net/browse/NIAD-2051

I've added the ticket number to the comment